source/features/mark-unread.tsx

```diff
@@ -409,7 +409,7 @@ async function init(): Promise<void> {
 			delegate('.js-delete-notification button', 'click', updateUnreadIndicator),
 			delegate('.js-mark-visible-as-read', 'submit', markVisibleNotificationsRead)
 		);
-	} else if (pageDetect.isPR() || pageDetect.isIssue()) {
+	} else if (pageDetect.isPRConversation() || pageDetect.isIssue()) {
```

Comment:
`markRead` needs to be called on all pages of PRs, not just the conversation view. Only the button needs to have this condition change

Reply:
Thanks, I don't pay attention to notifications at all, but this error pops up all the time. :)